tests/ui_tools/test_popups.py

```diff
@@ -179,6 +180,44 @@ def test_keypress_navigation(
 
         self.super_keypress.assert_called_once_with(size, navigation_key)
 
+    @pytest.mark.parametrize(
+        [
+            "contents",
+            "column_widths",
+            "expected_instance",
+        ],
+        [
+            (
+                [
+                    (
+                        "Viewing Actions",
+                        [
+                            ("Open in web browser", "[]"),
+                            ("Full rendered message", "[]"),
+                            ("Full raw message", "[]"),
+                        ],
+                    )
+                ],
+                [21, 24],
+                [Text, AttrWrap, AttrWrap, AttrWrap],
+            ),
+            ([("", "")], [21, 24], []),
```

Comment:
The `21` and `24` are the same in each test you have in this PR. Are they constants which must work with the test data in particular, or can these be constants in the test parameters/body, and simplify the test inputs?

Reply:
No they are not constants, I had added them as constants for temporary purposes just to test if the test worked. Changed it so that it's not hardcoded now.